### DISCUSSION TITLE
图标错误

### DISCUSSION THREAD

**Discussion by usermice** [hugo-next/hugo-theme-next#133] [Q&A]
![Snipaste_2023-12-26_21-49-45](https://github.com/hugo-next/hugo-theme-next/assets/86307502/13683e1a-ece4-42d1-b31a-3b53c731807d)
现在出现这种图标不显示错误， 再问下大佬啥时候给加个文档加密功能

**Comment by elkan1788** [comment]
这里使用的是FontAwsome字体库，需要检查一下网络问题。 关于加密的功能优先级别不高，现在主要是想把文档和代码结构整理好，希望能够吸引更多的开发者来共同维护。

**Comment by elkan1788** [comment]
刚检查了一下似乎是CDN服务的问题，升级一下字体库版本就好了，后面发个版本出来修复一下。
